src/views/Farms/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface TooltipProps {
+  content: React.ReactNode
+}
+
+const TooltipContent = styled.div`
+  background: ${({ theme }) => theme.tooltip.background};
+  padding: 16px;
+  border-radius: 16px;
+  color: ${({ theme }) => theme.tooltip.text};
+  width: max-content;
+  display: none;
+  padding: 16px;
+  max-height: 500px;
+  z-index: ${({ theme }) => theme.zIndices.modal};
+  position: absolute;
+  bottom: calc(100% + 16px);
+  transform: translate(34px, 0);
+  right: 0;
+  max-width: 246px;
```

Comment:
This number seems pretty arbitrary. It is okay for now but please try and void numbers like these.